### PR TITLE
Add network links and info bubble

### PR DESCRIPTION
**Pull request by chasefleming**
#658

### CODE REVIEW

File: docs/ecosystem/bridges.md [hunk 1]
```diff
@@ -7,6 +7,14 @@ sidebar_custom_props:
 
 # Bridges
 
+:::info
+## Coming Soon
+
+Details on bridges connecting EVM on Flow with other networks are currently in the works and will be shared soon. These bridges will facilitate seamless asset transfers and interoperability between EVM on Flow and various blockchains, enhancing the utility and reach of Flow's ecosystem.
```
Comment (jribbink) [inline docs/ecosystem/bridges.md:13]:
```suggestion
Details on bridges connecting Flow with other networks are currently in the works and will be shared soon. These bridges will facilitate seamless asset transfers and interoperability between Flow and various blockchains, enhancing the utility and reach of Flow's ecosystem.
```

Are we making this distinction in our messaging?  Or should it just be "Flow"? (I don't know the answer, just wanted to flag)

**Comment by Aliserag** [comment]
Screenshot 2024-04-05 at 9 49 18 AM

Can we hyperlink Testnet/Previewnet here.

**Comment by Aliserag** [comment]
Lets also add previewnet under contract browser and add a Support networks section there to make it consistent with the others.
Screenshot 2024-04-05 at 9 51 09 AM

**Comment by chasefleming** [comment]
> Screenshot 2024-04-05 at 9 49 18 AM
> 
> Can we hyperlink Testnet/Previewnet here.

That's what this PR does :)

**Pull request merged**